spaces/S000010/properties/P000201.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000010
+property: P000201
+value: true
+---
+
+{{S10}} has a particular point topology. The particular point is a generic point.
```

Comment:
should be single braces `{S10}`.  Or just say "This space has a particular point topology ..."